app/src/main/res/values/strings.xml

```diff
@@ -174,10 +174,10 @@
     <string name="your_media_is_available">Your media is available at</string>
 
     <string name="prefs_networking_title">Networking</string>
-    <string name="prefs_use_tor_title">Enable Onion Routing</string>
```

Comment:
Not sure Tor uses the term, "Onion Routing". Get someone who has access to their translation platform to check.